mitosheet/src/mito/components/toolbar/GraphTabs/ChartDesignTabContents.tsx

```diff
@@ -138,6 +149,7 @@ export const ChartDesignTabContents = (
                 />
             </Dropdown>
         </ToolbarButton>
+        <ToolbarButton action={props.actions.buildTimeActions[ActionEnum.Open_Graph_Select_Data_Taskpane]}/>
```

Comment:
It seems like the most sensible order would be: 

1. Select Data
2. Change Chart Type
3. Add Chart Element
4. Export 

What do you think?

<img width="654" alt="Screenshot 2024-02-05 at 4 21 35 PM" src="https://github.com/mito-ds/mito/assets/18709905/9a53c735-e386-409f-a203-017a2bc99a29">

Reply:
Good point! Done